scripts/build_requirements.py

```diff
@@ -4,11 +4,11 @@
 
 from utils import Downloadable
 
-POOL_URL = "https://deb.debian.org/debian/pool/main"
+POOL_URL = "http://prdownloads.sourceforge.net"
 
 SWIG = Downloadable(
-    name          = "swig-3.0.12.tar.gz",
-    unpacked_name = "swig-3.0.12",
-    url  = POOL_URL + "/s/swig/swig_3.0.12.orig.tar.gz",
-    hash = "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+    name          = "swig-4.0.1.tar.gz",
+    unpacked_name = "swig-4.0.1",
+    url  = POOL_URL + "/swig/swig-4.0.1.tar.gz",
+    hash = "7a00b4d0d53ad97a14316135e2d702091cd5f193bb58bcfcd8bc59d41e7887a9"
 )
```

Comment:
Changing from swig 3.0.x to 4.0.x _is_ an acceptable change.

Reply:
Correct, SWIG 4 is required to support Python 4.

Reply:
One more comment. The SWIG 3 did work before the change since we did not generate destructors (see below). And because of the it did not emit code to deal with Python garbage collector. Starting Python 3.8 the API for garbage collector has been changed and SWIG 3 generated code does not compile. So, this is why 4.x.